ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,65 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors
+using ITensorNetworks
+using ITensorNetworks: Index
```

Comment:
Let's list the functions and types we are using explicitly, i.e.:
```julia
using ITensors: Index, ...
using ITensorNetworks: ITensorNetworks, ITensorNetwork, ...
```
Also best to include a function or type from the module/package where it originated, i.e. `using ITensors: Index` instead of `using ITensorNetworks: Index`.

Reply:
Also, it seems like you may as well include more things into the namespace, like `ITensors.Algorithm`, `EinExprs.Optimizer`, etc. I wasn't clear on the logic of why certain things were included into the namespace while others were not.

Reply:
Unless they are indicating that they are low-level or private parts of the API, perhaps that is the case with `EinExprs.nargs`? If that's the case it seems worth prepending with the module name to remind us that the code is relying on internals of `EinExprs.jl`.

Reply:
`EinExprs.nargs` is "public" but not exported by default. I didn't find a use-case outside of these integrations, but it's solid and there shouldn't be any changes.

Reply:
Same thing with `EinExprs.Optimizer`: it's public but not exported.